src/main/resources/application.properties

```diff
@@ -158,7 +158,10 @@ refset.types.AssociationReference=900000000000521006|Content|c|targetComponentId
 refset.types.AttributeValue=900000000000480006|Content|c|valueId
 refset.types.Simple=446609009|Content||
 refset.types.Annotation=900000000000516008|Annotation|s|annotation
-refset.types.SimpleMap=900000000000496009|Map|s|mapTarget
+refset.types.SimpleMapToSCT=1187636009|Map|s|mapTarget
```

Comment:
The field in this row should be named `mapSource`.
The same is true for the SimpleMapCorrelationToSCT row.